tests/test_portchannel_global.rb

```diff
@@ -37,6 +38,11 @@ def cleanup
     config_no_warn "no port-channel load-balance #{ethernet}"
   end
 
+  def n3k_is_t2?
+    return unless /N3/ =~ node.product_id
+    Platform.chassis['descr'].include?('Nexus 31') ? true : false
```

Comment:
`'descr'` is a string. `include?` is for arrays. Also, the 31xx's use a string like `Nexus3000 C3132Q Chassis`